openedge.properties

```diff
@@ -16,3 +16,10 @@ defaults.mavenArtifactId=sonar-openedge-plugin
 2.24.2.description=Version 2.24.2
 2.24.2.sqVersions=[9.9,LATEST]
```

Comment:
Sorry, one more thing. Only one version can be marked compatible with `LATEST` at a time. You'll need to set this end range to 10.4 (since it was just released)

Reply:
Rebased commit, and changed value as requested.

Reply:
I'm still seeing 2.24.2 as compatible with `LATEST`...?

Reply:
Oops! Modification was still in the unstaged area... Fixed now.